### ISSUE TITLE
Original Display

### ISSUE THREAD

**Issue by Malvus524** [Bully85/Sovol-SV04-Klipper#23]
In the info you say something about an orig Display:
Do you mean the Original display from sovol or from the raspberry pi?
I ask because i cant get the Display from sovol to work.

**Comment by Bully85** [comment]
Which firmware do you install?

**Comment by Malvus524** [comment]
I was am using your old firmware but I have the configs updatet to the new ones but I can't get it to work.
I have updated start end macros, macros and index mode

**Comment by Bully85** [comment]
You must use the new firmware.bin all you need is in release 1.0.1

**Comment by Bully85** [comment]
https://github.com/Bully85/Sovol-SV04-Klipper/releases/tag/1.0.1

**Comment by Malvus524** [comment]
it still does not work. i have installed your new firmware on the mcu but i cant get the display to change the bin into a cur.
what am i doing wrong?

**Comment by Bully85** [comment]
Which config fiele you use?

**Comment by Bully85** [comment]
Do you change the repository?

**Comment by Malvus524** [comment]
The config file is the old one but I didn't change the repository

**Comment by Malvus524** [comment]
Do I have to install the new firmware to both the display and the MCU?

**Comment by Bully85** [comment]
for printer install: https://github.com/Bully85/Sovol-SV04-Klipper/releases/download/1.0.1/firmware.bin
for display: https://github.com/Bully85/DGUS-SV04/releases/download/1.0.0/DWIN_SET.rar

change the repository: https://github.com/Bully85/Sovol-SV04-Klipper#if-you-use-kiauh

the config: https://github.com/Bully85/Sovol-SV04-Klipper/releases/download/1.0.1/config.with.and.without.Display.rar

than import your slicer profiel cura(https://github.com/Bully85/Sovol-SV04-Klipper/releases/download/1.0.1/Cura.Profile.include.startcode_md.rar) or Prusa (https://github.com/Bully85/Sovol-SV04-Klipper/releases/download/1.0.1/Prusaslicer.Profile.include.startcode_md.rar)

now all must run

**Comment by Malvus524** [comment]
Thank you very much

**Comment by scuba1976** [comment]
I install original display and it works. It is normal the there isn't "print" command?

**Comment by Bully85** [comment]
Yes at the moment. You can't start a print with the display. At the moment only for temp and cancel and secure stop
